sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -37,6 +39,27 @@ import org.apache.spark.sql.types.StructType
 class JoinSuite extends QueryTest with SharedSparkSession {
   import testImplicits._
 
+  private def checkCleanupResourceTriggered(plan: SparkPlan): ArrayBuffer[SortExec] = {
```

Comment:
how about `attachCleanupResourceChecker`?

Reply:
and this method should return `Unit`

Reply:
Ah yes, sure, done in b41f33a.